bin/install

```diff
@@ -149,4 +149,18 @@ get_download_url_for_version() {
   echo "https://repo.hex.pm/builds/elixir/${version}.zip"
 }
 
+run_default_mix_commands() {
+  local install_path=$1
+  local default_mix_commands="${HOME}/.default-mix-commands"
+
+  if [ ! -f "$default_mix_commands" ]; then return; fi
+
+  while read -u 7 -ra command; do
```

Comment:
What is the significance of `-u 7` here?

Reply:
It doesn't seem to be significant. I'll remove it and I'll fix the linting feedback. Thanks for taking a look!

Reply:
Oops, I was wrong, on line 163 I'm reading the default mix commands file and passing that to a file descriptor no 7. `read -u 7` reads from that file descriptor. If I don't do it like this running the first command will break out of my loop since it probably also reads or writes from the 'current' file descriptor.

Reply:
I don't fully understand what the file descriptor 7 represents, or if it will ever interfere with anything else. But the code works for me and CI tests pass so I'll merge and we can re-evaluate this later if it causes a problem.